ingestion_tools/scripts/common/config.py

```diff
@@ -16,28 +16,6 @@
     BaseImporter = "BaseImporter"
 
 
-APPEND_STATIC_CONFIG: str = """
```

Comment:
is this related to the PR?

Reply:
Yeah, I cleaned up to make this more intutive as I was updating the default config value to be conditional.